app.js

```diff
@@ -291,29 +283,13 @@ api.on('connection', function (spark) {
 
   spark.on('history', function (data) {
```

Comment:
why do we ignore history here?

Reply:
we could, but might have a side effect if we can't update ethstats.go/`reportHistory()`